### PR TITLE
Be clearer in the editors page about what the purpose of each plugin is

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
and if you need multiple, one of, etc

### CODE REVIEW

File: guides/release/code-editors/index.md [hunk 1]
```diff
@@ -37,29 +45,53 @@ Both editors share a range of cross-compatible extensions listed below.
...
-[coc-ember](https://github.com/NullVoxPopuli/coc-ember) -
+With the ember plugin [coc-ember](https://github.com/NullVoxPopuli/coc-ember) -
 Ember.js language server extension including useful configuration instructions.
 
-[nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter) -
-high-fidelity static highlighting. With `ensure_installed = { 'glimmer' }`, this
-also includes support for embedded templates in JavaScript and typescript.
+[Conquer for Completion (COC) for Neovim](https://github.com/neoclide/coc.nvim) -
+An Intellisense engine which takes control over all linting, hinting, and language-server integration.
+
```
Comment (jenweber) [inline guides/release/code-editors/index.md:83]: This is a repeat of line 76. Can you delete this or the previous mention above?

File: guides/release/code-editors/index.md [hunk 2]
```diff
@@ -37,29 +45,53 @@ Both editors share a range of cross-compatible extensions listed below.
...
+To get embedded highlighting will involve these additional plugins:
+- [vim-javascript](https://github.com/pangloss/vim-javascript)
+- [vim-js-pretty-template](https://github.com/Quramy/vim-js-pretty-template)
 
-[Ember Tools](https://github.com/AndrewRadev/ember_tools.vim) -
-Various tools for working with Ember.js projects.
+
+### Language Server
+
+Only one of these solutions should be used at a time
```
[the remark this answers is not in the document]
Comment (NullVoxPopuli) [reply guides/release/code-editors/index.md]: thanks!

File: guides/release/code-editors/index.md (hunk 1 above)
Comment (NullVoxPopuli) [reply guides/release/code-editors/index.md]: oops. thanks!

**Review by jenweber — APPROVED** [review]

**Comment by jenweber** [comment]
Thanks again for these updates!

**Pull request merged**